### ISSUE TITLE
Failed to activate

### ISSUE THREAD

**Issue by Kentoseth** [isamert/gedi#3]
OS: ubuntu 14.04

Install method:

`apt-get install python-jedi`

(python-jedi (0.7.0-1))

Placed 2 scripts in path `/.local/share/gedit/plugins/`

script-names: 
- gedi.py
- gedi.plugin

> The plugin 'Gedi' could not be loaded
> An error occurred: Failed to load

How do I debug this?

**Comment by isamert** [comment]
Which version of Gedit are you running? Because this plugin only supports Gedit version 3 and higher.

**Comment by Kentoseth** [comment]
The standard Ubuntu 14.04 version:

`3.10.4`

**Comment by isamert** [comment]
Sorry for the late response, somehow my notifications got broken.

I have actually no idea what causes this. Is `python3` command working?

**Comment by djbucci** [comment]
Make sure you pip3 install jedi. I had to do this, since I'm one of the few still stuck in the past that works 2.7.

**Comment by fel-ap** [comment]
Hi, I'm also having issues setting up Gedi. The plugin shows up in gedit's plugin list and I managed to load it successfully. However whenever I type some code, I repeatedly get this error in the linux terminal :

File "/usr/lib/i386-linux-gnu/gedit/plugins/gedi.py", line 69, in do_match
    _, iter = context.get_iter()
TypeError: 'TextIter' object is not iterable

This message shows up after every character I type with my keyboard. Can you please help me out with  this ?

Thank you.

**Comment by isamert** [comment]
@fel-ap I commited a fix, can you try it and tell me if it works? I can't test it right now because I'm not on a GNOME environment.

**Comment by fel-ap** [comment]
@isamert Yes it seems to be working thanks ! If anything else shows up, I will let you know.
